pandas/core/indexes/base.py

```diff
@@ -3743,7 +3743,9 @@ def drop(self, labels, errors='raise'):
         return self.delete(indexer)
 
     @Appender(base._shared_docs['unique'] % _index_doc_kwargs)
-    def unique(self):
+    def unique(self, level=None):
+        if level not in {0, self.name, None}:
+            raise ValueError("Level {} not found".format(level))
```

Comment:
test for this?

Reply:
They are [here](https://github.com/pandas-dev/pandas/pull/17897/files#diff-9ae0a94014431c93e23da48433811c49R342)

Reply:
is the doc string for unique updated?

Reply:
No, because you never replied to [my question](https://github.com/pandas-dev/pandas/pull/17897#discussion_r145226159)

Reply:
not sure what the question is